internal/compiler/parse_test.go

```diff
@@ -0,0 +1,104 @@
+package compiler
```

Comment:
I take a bit of a different approach to testing sqlc, opting mainly for end-to-end tests that don't verify internal behavior. Can you take these tests and move them into internal/endtoend/testdata/order_by_non_existing_column?

Reply:
That I can do. We actually only need these tests for order by. The group by tests are in internal/endtoend/testdata/invalid_group_by_reference already